llvm_jni/dist/native/nballerina.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+set -e
+
+print_usage_and_exit() {
+    echo "Usage: $0 <filename.bal> [--target aarch64]"
+    exit "$1"
+}
+
+if [ $# -lt 1 ]; then
+    print_usage_and_exit 1
+fi
+
+if [ "$1" = "--help" ] || [ "$1" = "-h" ]; then
+    print_usage_and_exit 0
+fi
+
+scriptDir=$(dirname "$0")
+src="$1"
+buildDir="$(pwd)/build"
+rm -rf "$buildDir"
```

Comment:
Doing rm -fr on a directory in the current directory seems a bit drastic. What if the user already had created a `build`` directory?

Reply:
Fixed